### ISSUE TITLE
Using default fonts

### ISSUE THREAD

**Issue by mitar** [fuzhenn/node-svg2img#6]
It seems default fonts canvas 1 is using are pretty limited and do not really match those available in the browsers, even on the same system. So SVG rendered in the browser is different from a PNG rendered with this library it SVG contains text.

**Comment by fuzhenn** [comment]
[node-canvas](https://github.com/Automattic/node-canvas)'s parsing of ```font-family```is stricter than in browser: in node-canvas, ```font-family``` needs to be double quoted.
e.g.
```bold 12px/20px arial,sans-serif ``` is OK in browser, but with node-canvas it has to be 
```bold 12px/20px "arial,sans-serif"```

You may give a try.

**Comment by fuzhenn** [comment]
In my experience, node-canvas's font rendering is always different from browser. I guess it's caused by [cairo](https://www.cairographics.org/), the 2d graphics lib used by node-canvas.

**Comment by mitar** [comment]
Hm, I am using: `font-family="Helvetica, Arial, Lucida Grande, sans-serif"`. So what I think it would be interesting if node-canvas 2.0 `registerFont` would be passed through so that I can register additional fonts. (Once this package will be using 2.0.)

**Comment by fuzhenn** [comment]
I have made a quick check of canvg's source codes, I found the font style is not double quoted in canvg.

I have fixed it here : https://github.com/fuzhenn/canvg/commit/623676abbb08dc7d4d2c26db3da922c7e89ac8bf

You can update your canvg and make another try.

**Comment by mitar** [comment]
Hm, I tried it but in my particular case I do not see a difference. It might that `Helvetica` is simply not available through Cairo?

**Comment by fuzhenn** [comment]
Yeah, i guess, ```Helvetica``` font is a system-dependent one. I have tested some other fonts (on Windows), and they are all correctly rendered.

And I'll update node-canvas to 2.0 once it's stable and released, maybe it can fix it.

**Comment by lookatitude** [comment]
I had the same problem on a project runing on AWS lambda, se solution was:
1 - add the fonts to your OS - in our case we used a layer to load the fonts into the aws image.
2 - before calling svg2img register your fonts in Canvas:
`const Canvas = require('canvas')
Canvas.registerFont('/var/task/fonts/Archivo-Regular.ttf', {family: 'Archivo'});`
